config/main.py

```diff
@@ -1450,15 +1449,15 @@ def reload(db, filename, yes, load_sysinfo, no_service_restart, disable_arp_cach
     if not force and not no_service_restart:
         if _is_system_starting():
             click.echo("System is not up. Retry later or use -f to avoid system checks")
-            return
+            sys.exit(1)
 
         if not _delay_timers_elapsed():
             click.echo("Relevant services are not up. Retry later or use -f to avoid system checks")
-            return
+            sys.exit(1)
```

Comment:
>1 [](http://example.com/codeflow?start=21&length=1)

Better use different return codes for different reasons.

Reply:
All these three error conditions are identical which are meant to signal config reload is not ready to be executed. The appropriate error message would give user the hint why it is not ready. I believe since they are for the same behavior the same exit code should be used.
Please let me know your thoughts.

Reply:
Could you define a constant for "config reload is not ready to be executed" return code?

Reply:
Done